src/report/terminal.rs

```diff
@@ -98,28 +140,23 @@ pub fn report_opts(
     } = r#match;
 
     let highlighter = create_highlighter(opts.color)?;
-    let w;
-    let hl;
-    let (bold, warn, highlight, reset) = if opts.color {
-        w = format!("{COLOR_BOLD}{COLOR_RED}");
-        hl = format!("{COLOR_BOLD}{COLOR_BLUE}");
-        (COLOR_BOLD, w.as_str(), hl.as_str(), COLOR_RESET)
-    } else {
-        ("", "", "", "")
-    };
+    let (bold, warn, highlight, reset) = get_format_strings(opts.color);
+
+    let escaped_lint_name = escape_html(lint_name);
+    let escaped_message = escape_html(message);
+    let escaped_path = escape_html(&path.display().to_string());
```

Comment:
This is probably the most "unfortunate" since we repurpose the terminal file.
The alternative would be a `html.rs` file but it would look nearly identical except for the escaping.